### PR TITLE
90%: Only dim input text when disabled

### PR DESCRIPTION
**Pull request by pads**
Changes a disabled input from this:

wrong

To this:

right

### CODE REVIEW

File: assets/stylesheets/bootstrap/_forms.scss [hunk 1]
```diff
@@ -143,7 +143,6 @@ output {
   // HTML5 says that controls under a fieldset > legend:first-child won't be
   // disabled if the fieldset is disabled. Due to implementation difficulty, we
   // don't honor that edge case; we style them as disabled anyway.
-  &[disabled],
   &[readonly],
```
Comment (pads) [inline assets/stylesheets/bootstrap/_forms.scss]: How does `readonly` differ from `disabled`? Should it also have the same changes?

**Comment by danielmatthew** [comment]
I prefer this - quite why disabled inputs were blanked out previously, I don't really know. 

However, other than colour contrast, is there any other indication that the field is disabled?

File: assets/stylesheets/bootstrap/_forms.scss (hunk 1 above)
Comment (danielmatthew) [inline assets/stylesheets/bootstrap/_forms.scss]: Different semantics to the user and AT.

**Comment by pads** [comment]
When you hover over it you get the disabled cursor effect. I personally thing the background colour should be different because it's too subtle. One for @jeremybaines.

File: assets/stylesheets/bootstrap/_forms.scss (hunk 1 above)
Comment (pads) [inline assets/stylesheets/bootstrap/_forms.scss]: In the example I am using this for, you tick a check box and it copies values from one form to the other. The copied to location becomes disabled until you un-check the box. Would `readonly` still be suitable here?

Comment (pads) [inline assets/stylesheets/bootstrap/_forms.scss]: I'll use `readonly` - what you say applies to the example above.

**Comment by pads** [comment]
@danielmatthew are you happy for this to be merged?

**Comment by danielmatthew** [comment]
@pads I'd prefer the colour sorted before it goes down, but if it's holding things up, feel free to bump the version and release!

**Comment by pads** [comment]
It's not yet but will probably do early next week. Which colour needs addressing?

**Pull request merged**